ee/modules/160-multitenancy-manager/templates/user-resources/default-project-template.yaml

```diff
@@ -0,0 +1,114 @@
+  namespaceMetadata:
+    annotations:
+      extended-monitoring.deckhouse.io/enabled: ""
+    labels:
+      created-from-project-type: default-project-template
+  valuesSchema:
+    requests:
+      type: object
+      properties:
+        cpu:
+          oneOf:
+            - type: number
+              format: int
+            - type: string
+          pattern: "^[0-9]+m?$"
+        memory:
+          oneOf:
+            - type: number
+              format: int
+            - type: string
+          pattern: '^[0-9]+(\.[0-9]+)?(E|P|T|G|M|k|Ei|Pi|Ti|Gi|Mi|Ki)?$'
+        storage:
+          type: string
+          pattern: '^[0-9]+(\.[0-9]+)?(E|P|T|G|M|k|Ei|Pi|Ti|Gi|Mi|Ki)?$'
+    limits:
+      type: object
+      properties:
+        cpu:
+          oneOf:
+            - type: number
+              format: int
+            - type: string
+          pattern: "^[0-9]+m?$"
+        memory:
+          oneOf:
+            - type: number
+              format: int
+            - type: string
+          pattern: '^[0-9]+(\.[0-9]+)?(E|P|T|G|M|k|Ei|Pi|Ti|Gi|Mi|Ki)?$'
+  template: |
+    ---
+    # Max requests and limits for resource and storage consumption for all pods in a namespace.
+    # Refer to https://kubernetes.io/docs/concepts/policy/resource-quotas/
+    apiVersion: v1
+    kind: ResourceQuota
+    metadata:
+      name: all-pods
+    spec:
+      hard:
+        {{ with .templateValues.requests.cpu }}requests.cpu: {{ . }}{{ end }}
+        {{ with .templateValues.requests.memory }}requests.memory: {{ . }}{{ end }}
+        {{ with .templateValues.requests.storage }}requests.storage: {{ . }}{{ end }}
+        {{ with .templateValues.limits.cpu }}limits.cpu: {{ . }}{{ end }}
+        {{ with .templateValues.limits.memory }}limits.memory: {{ . }}{{ end }}
+    ---
+    # Max requests and limits for resource consumption per pod in a namespace.
+    # All the containers in a namespace must have requests and limits specified.
+    # Refer to https://kubernetes.io/docs/concepts/policy/limit-range/
+    apiVersion: v1
+    kind: LimitRange
+    metadata:
+      name: all-containers
+    spec:
+      limits:
+        - max:
+            {{ with .templateValues.limits.cpu }}cpu: {{ . }}{{ end }}
+            {{ with .templateValues.limits.memory }}memory: {{ . }}{{ end }}
+          maxLimitRequestRatio:
+            cpu: 1
+            memory: 1
+          type: Container
+    ---
+    # Deny all network traffic by default except namespaced traffic and dns.
+    # Refer to https://kubernetes.io/docs/concepts/services-networking/network-policies/
+    kind: NetworkPolicy
+    apiVersion: networking.k8s.io/v1
+    metadata:
+      name: deny-all-except-current-namespace
+    spec:
+      podSelector:
+        matchLabels: {}
+      policyTypes:
+        - Ingress
+        - Egress
+      ingress:
+        - from:
+            - namespaceSelector:
+                matchLabels:
+                  kubernetes.io/metadata.name: "{{ .projectName }}"
```

Comment:
With the current ingress policy, Prometheus will not be able to scrape metrics. Let's add the rule for it:
```yaml
- namespaceSelector:
    matchLabels:
       kubernetes.io/metadata.name: "d8-monitoring"
   podSelector:
     matchLabels:
       app: prometheus
```

Reply:
implemented